src/sql/workbench/services/accountManagement/browser/accountPickerImpl.ts

```diff
@@ -153,11 +193,30 @@ export class AccountPicker extends Disposable {
 			DOM.show(this._refreshContainer);
 		} else {
 			DOM.hide(this._refreshContainer);
+
+			if (account.properties.tenants?.length > 1) {
+				if (this._tenantContainer?.parentElement?.parentElement) {
+					DOM.show(this._tenantContainer.parentElement.parentElement);
```

Comment:
This is a pretty weird thing to do - I wouldn't expect a component to be manipulating the DOM of its ancestors like this...why not just show/hide the both the root and account containers? (I think that's what you're trying to do right?)

Reply:
Trying to hide the entire tenant label

Reply:
I agree as well, I could put some more time refactoring how this is designed but honestly I'm too worried about breaking some other things.